app/controllers/application_controller/explorer.rb

```diff
@@ -169,6 +169,7 @@ def tree_select(node_info = false)
   # Accordion selected in explorer
   def accordion_select(node_info = false)
     @lastaction = "explorer"
+    @sb[:action] = nil
```

Comment:
assuming `tree_select` is doing `if @sb` for a reason, should this do the same?
(As in, is it definitely impossible to get to accordion_select without having sb initialized?)

Reply:
Fixed